Providers/Scripts/2.4x-2.5x/Scripts/nxOMSContainers.py

```diff
@@ -0,0 +1,284 @@
+#!/usr/bin/env python
+#============================================================================
+# Copyright (C) Microsoft Corporation, All rights reserved.
+#============================================================================
+
+from contextlib import contextmanager
+
+import os
+import sys
+import imp
+import re
+import pwd
+import grp
+import codecs
+import shutil
+
+protocol = imp.load_source('protocol', '../protocol.py')
+nxDSCLog = imp.load_source('nxDSCLog', '../nxDSCLog.py')
+
+LG = nxDSCLog.DSCLog
+
+MODULE_RESOURCE_DIR ='/opt/microsoft/omsconfig/modules/nxOMSContainers/DSCResources/MSFT_nxOMSContainersResource/containers'
+
+PLUGIN_PATH = '/opt/microsoft/omsagent/plugin/'
+CONF_PATH = '/etc/opt/microsoft/omsagent/<WorkspaceId>/conf/omsagent.d'
+REG_PATH = '/etc/opt/omi/conf/omiregister/root-cimv2'
+LIB_PATH = '/opt/microsoft/docker-cimprov/lib'
+YAML_PATH = '/var/opt/microsoft/docker-cimprov/state'
+
+class IOMSAgent:
+    def restart_oms_agent(self):
+        pass
+    
+class OMSAgentUtil(IOMSAgent):
+    def restart_oms_agent(self):
```

Comment:
Like nxOMSPlugin, with multi-homing, this restart command should only apply to the workspace ID that you're operating on. This is for targeting and efficiency.

Reply:
You can refer https://github.com/Microsoft/PowerShell-DSC-for-Linux/blob/master/Providers/Scripts/3.x/Scripts/nxOMSAuditdPlugin.py